<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScene</class>
 <widget class="QWidget" name="MainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	font: 20pt &quot;Menlo&quot;;
}

QPushButton
{
	font: 20pt &quot;FontAwesome&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="toolBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="menuButton">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="undoButton">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="clickedStatus" stdset="0">
         <string>no</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="giveupButton">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="clickedStatus" stdset="0">
         <string>no</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="centralSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="remainingTime">
        <property name="font">
         <font>
          <family>Menlo</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 24px;</string>
        </property>
        <property name="text">
         <string>00:20</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="transferButton">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="isWhite" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="isWhite" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="FiveBoardView" name="boardView">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FiveBoardView</class>
   <extends>QGraphicsView</extends>
   <header>fiveboardview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
